server/routes/profile.js

```diff
@@ -1,25 +1,36 @@
 const express = require('express');
 
-const createProfileRouter = () => {
+const createProfileRouter = ({ offenderService }) => {
   const router = express.Router();
 
+  const getPersonalisation = async user => {
+    const events = await offenderService.getEventsForToday(user);
+
+    const timetableError = events?.error;
```

Comment:
events can never be null so won't need the `?.`

As Jon suggested this could probably be moved into the line below: 
`const { morning, afternoon, evening, error: timeTableError } = events || {};`

Reply:
this is now addressed